src/test/java/com/powsybl/openloadflow/sensi/dc/DcSensitivityAnalysisTest.java

```diff
@@ -884,4 +887,97 @@ void nonImpedantBranchTest() {
         assertEquals(-0.6666666, result.getBranchFlow1SensitivityValue("glsk", "L2"), LoadFlowAssert.DELTA_POWER);
         assertEquals(66.6666, result.getBranchFlow1FunctionReferenceValue("L2"), LoadFlowAssert.DELTA_POWER);
     }
+
+    @Test
+    void testConfiguredBusFactor() {
+        Network network = EurostagTutorialExample1Factory.create();
+        runAcLf(network);
+
+        SensitivityAnalysisParameters sensiParameters = createParameters(true, "VLLOAD_0");
+
+        List<SensitivityFactor> factors = List.of(createBranchFlowPerInjectionIncrease("NHV1_NHV2_1", "GEN"),
+                                                  new SensitivityFactor(SensitivityFunctionType.BRANCH_ACTIVE_POWER_1,
+                                                                        "NHV1_NHV2_1",
+                                                                        SensitivityVariableType.INJECTION_ACTIVE_POWER,
+                                                                        "NGEN",
+                                                                        false,
+                                                                        ContingencyContext.all()));
+
+        SensitivityAnalysisResult result = sensiRunner.run(network, factors, Collections.emptyList(), Collections.emptyList(), sensiParameters);
+
+        assertEquals(2, result.getValues().size());
+        assertEquals(result.getBranchFlow1SensitivityValue("GEN", "NHV1_NHV2_1"),
+                     result.getBranchFlow1SensitivityValue("NGEN", "NHV1_NHV2_1"),
+                     LoadFlowAssert.DELTA_POWER);
+    }
+
+    @Test
+    void testConfiguredBusInvalidFactor() {
```

Comment:
I am not sure to understand why it is an invalide factor and when the sensitivity value is equal to zero. Thanks!